calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -119,15 +123,18 @@
 	
 	private static final String EVENT_URL_PATTERN = "%s?eventReference=%s&panel=Main&sakai_action=doDescription&sakai.state.reset=true";
 
-   private DocumentBuilder docBuilder = null;
+   	private DocumentBuilder docBuilder = null;
    
-   private static final ResourceLoader rb = new ResourceLoader("calendar");
+   	private static final ResourceLoader rb = new ResourceLoader("calendar");
    
-   private ContentHostingService contentHostingService;
+   	private ContentHostingService contentHostingService;
 
-   private ExternalCalendarSubscriptionService externalCalendarSubscriptionService;
+	@Setter
+	private CourseManagementService courseManagementService;
 
-   PDFExportService pdfExportService;
+   	private ExternalCalendarSubscriptionService externalCalendarSubscriptionService;
+
+   	PDFExportService pdfExportService;
```

Comment:
I know it's unrelated to the commit, but can we make this private?

Reply:
Agreed